### PR TITLE
Medical Pouches - Public Medical Supply Buff - More Medical Supplies!

### PR DESCRIPTION
**Pull request by WickedCybs**
With the setting change to a ship, I've found we're still lacking in the medical supply department for the crew overall. It makes expeditions more deadly than they have to be and in terms of quality of life it's just pretty annoying to open a medical closet and get one gauze, one ointment. 

What this PR sets out to do is buff medical supplies that the crew can generally access by a very large margin. They'll provide more supplies, more kits and a new medical item, the medical pouches. These pouches have a good amount of liquid medicine in autoinjectors and pills and are designed to keep someone alive long enough so that medical can receive them. Especially good is that the autoinjectors means they'll still come in handy in space. During expeditions, taking off suits can be a luxury at times.

I'll update the main body of this post as things get set in stone and I've completed what I've set out to do. 

Also, all credit to Bay for these pouches, as their implementation is the basis on how I'm doing this one.

### CODE REVIEW

**Comment by WickedCybs** [comment]
!wip

**Comment by WickedCybs** [comment]
Current problems are

The cross overlay only shows up once I remove an item from the pouch
2022JuneMonday13041839

Once the pouch is open, you can never look at its contents again.

File: code/game/objects/items/weapons/storage/med_pouch.dm [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+	8) Stay in place once they respond."
+
+/obj/item/storage/med_pouch/Initialize()
+	. = ..()
+	name = "emergency [injury_type] pouch"
+	make_exact_fit()
+	for(var/obj/item/reagent_containers/pill/P in contents)
+		P.color = color
+	for(var/obj/item/reagent_containers/hypospray/autoinjector/A in contents)
+		A.update_icon()
```
Comment (alsoandanswer) [inline code/game/objects/items/weapons/storage/med_pouch.dm:36]: Shove in an update_icon() again here so it'll add the cross overlay.

The A.update_icon() is for the autoinjector.

File: html/changelogs/wickecybs_medical.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+author: WickedCybs
+
+delete-after: True
+
+changes:
+  - rscadd: "Mention the pouches."
```
Comment (alsoandanswer) [inline html/changelogs/wickecybs_medical.yml:6]: I think you forgot to update this.

Comment (SleepyGemmy) [reply html/changelogs/wickecybs_medical.yml:6]: PR is marked WIP.

File: code/game/objects/items/weapons/storage/med_pouch.dm (hunk 1 above)
Comment (WickedCybs) [reply code/game/objects/items/weapons/storage/med_pouch.dm:36]: This fixed everything, thank you Wezzy.

File: html/changelogs/wickecybs_medical.yml (hunk 2 above)
Comment (WickedCybs) [reply html/changelogs/wickecybs_medical.yml:6]: I'll be putting in more details once ironed out certain bits of the PR and have it ready to go.

**Pull request closed without merging**